packages/devtools_app/lib/src/service_extensions.dart

```diff
@@ -93,6 +93,18 @@ final debugAllowBanner = ToggleableServiceExtensionDescription<bool>._(
   gaItem: ga.debugBanner,
 );
 
+final invertOversizedImages = ToggleableServiceExtensionDescription<bool>._(
```

Comment:
It isn't great but material design has an invert colors icon.
https://material.io/resources/icons/?icon=invert_colors&style=baseline

Reply:
I was able to find a reasonable 'image' icon in the set of material ones.